ci/docker/runtime_functions.sh

```diff
@@ -22,10 +22,13 @@
 
 set -ex
 
+# lower priority of build and test tasks
+renice -n 19 -p $$
```

Comment:
As discussed before, I don't like changing priorities.

Reply:
Let's talk about this offline then.